### ISSUE TITLE
Waiting for the Database to be available 1/10 getting bigger then 10

### ISSUE THREAD

**Issue by zewas-digital** [phpList/phplist-docker#24]
I am trying to install phpList with Docker.
I set the .env variables as shown in [link](https://github.com/phpList/phplist-docker/blob/master/.env-dist) with my own values.

When I deploy the container, it deploys fine, but when I try to access the application, I get a bad gateway error.

When I look in the logs, the message appears: `Waiting for database to be available - 1/10` appears. 
Now this message keeps appearing and I am currently at `Waiting for database to become available - 66/10`.
Do I need to add a database container myself or have I forgotten something?

**Comment by michield** [comment]
Yes, you need to give a valid end-point for a DB connection. That can be another container, or a different server. Interesting it goes over 10, I will look into that. The idea is that if it didn't work in 10 steps, it's unlikely to work, so it should stop after 10 iterations. When you bring up two containers, it can take a little while for the DB to be ready, which is why this delay is there.

**Comment by dapasa** [comment]
In docker, I have fixed it using this env variables: DB_HOST, DB_USER, DB_PASSWORD, DB_NAME. In some way the config.php is taked from container phat /etc/phplist, but that config.php is using the mentioned variables instead of the ones into the .env file.
